graphics/include/ignition/common/ColladaExporter.hh

```diff
@@ -47,6 +49,10 @@ namespace ignition
       public: virtual void Export(const Mesh *_mesh,
           const std::string &_filename, bool _exportTextures = false);
 
+      public: virtual void Export(const Mesh *_mesh,
+          const std::string &_filename, bool _exportTextures,
+          std::vector<math::Matrix4d> &_submeshToMatrix);
```

Comment:
`_submeshToMatrix` is not being modified by the function right? Can you use `const &`?